sr-eval/README.md

```diff
@@ -0,0 +1,36 @@
+# Scene Recognition Evaluation
+This involves evaluating the results of the scenes-with-text classification task.
+The goal is to have a simple way of comparing different results from SWT.
```

Comment:
Since SWT returns two types of annotations (time point and time frame), let's make it more clear that in this subdirectory, only point-wise annotations are dealt with.